templates/digital-asset-template/frontend/App.tsx

```diff
@@ -14,170 +28,192 @@ const COLLECTION_DESCRIPTION = `Step into the enchanting world of "Forest Friend
 
 function App() {
   return (
-    <main className="flex flex-col gap-6">
+    <div>
       {/* Nav Bar */}
-      <nav className="navbar-container px-6 flex justify-between">
-        <p className="navbar-text">Forest Friends</p>
+      <nav className="navbar-container px-6 pt-6 flex justify-between">
+        <p className="display">Forest Friends</p>
         <div>
           <WalletSelector />
         </div>
       </nav>
 
-      {/* Hero */}
-      <section className="hero-container flex gap-6 px-6">
-        <img src={IMG()} />
-        <div>
-          <h1 className="title-md">{COLLECTION_NAME}</h1>
-          {config.socials && (
-            <ul className="flex gap-4">
-              {config.socials.twitter && (
-                <li>
-                  <img />
-                </li>
-              )}
-              {config.socials.discord && (
-                <li>
-                  <img />
-                </li>
-              )}
-            </ul>
-          )}
-          <p className="body-sm">{COLLECTION_DESCRIPTION}</p>
-          <div className="flex justify-between items-center">
-            <div className="flex gap-4">
-              <Input />
-              <Button>Mint</Button>
-            </div>
+      <main className="flex flex-col gap-10 md:gap-16 mt-6">
+        {/* Hero */}
+        <section className="hero-container flex flex-col md:flex-row gap-6 px-6">
```

Comment:
I think we can get each section into its own components - would be easier for devs to remove/add components

Reply:
Yep! I'm keeping everything colocated at the moment, when everything is mostly in place I am going to componentize everything and tidy things up